src/sentry/incidents/endpoints/serializers.py

```diff
@@ -41,6 +41,7 @@
 from sentry.models.organizationmember import OrganizationMember
 from sentry.models.team import Team
 from sentry.models.user import User
+from sentry.shared_integrations.exceptions import ApiRateLimitedError
```

Comment:
The except statements in this file ensure that we catch rate-limiting errors on creation and update, for metric alerts.